src/main/java/net/reini/rabbitmq/cdi/MessageConverter.java

```diff
@@ -0,0 +1,25 @@
+package net.reini.rabbitmq.cdi;
+
+/**
+ * Transform a object in a array de bytes to send a message.
+ * 
+ * @author André Ignacio
+ */
+public interface MessageConverter {
```

Comment:
I would like to have a generic type `M` of a converter like: `public interface MessageConverter<M>` for better type savety.